nyan/ast.cpp

```diff
@@ -885,7 +877,7 @@ ASTError::ASTError(const std::string &msg,
 	if (add_token) {
 		std::ostringstream builder;
 		builder << msg << ": "
-		        << token_type_str(token.get_type());
+				<< token_type_str(token.get_type());
```

Comment:
Isn't that because `<<` is technically a line continuation? The documentation says:

> `UT_AlignWithSpaces` (in configuration: `AlignWithSpaces`) Use tabs for line continuation and indentation, and spaces for alignment.

Reply:
it's a line continuation that should align to the previous `<<`. i thought we found a solution for this already to format correctly.
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignoperands

but it seems to be weird anyway, even with always-tab (which we don't have):
https://github.com/llvm/llvm-project/issues/59797